guava/src/com/google/common/io/LineReader.java

```diff
@@ -14,19 +14,29 @@
 
 package com.google.common.io;
 
-import static com.google.common.base.Preconditions.checkNotNull;
```

Comment:
Did you chance the import order accidentally? [Google Java Styleguide](https://google.github.io/styleguide/javaguide.html#s3.3-import-statements)

Reply:
yes, fixed.